wrappers/java/build.gradle

```diff
@@ -1,11 +1,44 @@
-plugins {
-    id 'java'
-}
+apply plugin: 'java'
+apply plugin: 'application'
 
 version '1.0-SNAPSHOT'
 
 sourceCompatibility = 1.8
 
+mainClassName = 'tech.pegasys.era.tools.EraNetworkConfig'
```

Comment:
The "EraNetworkConfig" script is getting treated as the main class of the ERA. Ideally it would have its script be created in the same way as the other 2 scripts, and the default main script creation task would be disabled.  

Alternatively, make the main class be the EraViewer, since if it gets run accidentally its effect will not be destructive; while the other scripts can end up sending transactions to the blockchain.